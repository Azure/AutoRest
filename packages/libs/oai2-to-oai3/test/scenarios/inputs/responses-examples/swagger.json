{
  "swagger": "2.0",
  "info": {
    "version": "1.0",
    "title": "test",
    "description": "Test"
  },
  "produces": ["application/json"],
  "consumes": ["application/json"],
  "paths": {
    "/invalid-response-examples": {
      "get": {
        "operationId": "InvalidResponseExamples",
        "consumes": [],
        "produces": ["application/json", "text/json"],
        "responses": {
          "200": {
            "description": "OK",
            "schema": { "type": "string" },
            "examples": {
              "application/json": "This example is valid  as 'application/json' is defined in the global produces",
              "text/json": "This example is valid  as 'text/json' is defined in the operation produces",
              "application/xml": "This example is invvalid  as 'application/xml' is NOT defined in the produces. It shouldn't be carried to oai3",
              "unknown": "This example is invvalid as 'unknown' is NOT defined in the produces. It shouldn't be carried to oai3"
            }
          }
        }
      }
    }
  },
  "definitions": {}
}
