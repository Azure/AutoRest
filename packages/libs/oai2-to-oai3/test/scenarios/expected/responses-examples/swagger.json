{
  "servers": [],
  "openapi": "3.0.0",
  "info": {
    "version": "1.0",
    "title": "test",
    "description": "Test"
  },
  "paths": {
    "/invalid-response-examples": {
      "get": {
        "operationId": "InvalidResponseExamples",
        "responses": {
          "200": {
            "description": "OK",
            "content": {
              "application/json": {
                "schema": {
                  "type": "string"
                },
                "examples": {
                  "response": {
                    "value": "This example is valid  as 'application/json' is defined in the global produces"
                  }
                }
              },
              "text/json": {
                "schema": {
                  "type": "string"
                },
                "examples": {
                  "response": {
                    "value": "This example is valid  as 'text/json' is defined in the operation produces"
                  }
                }
              }
            }
          }
        }
      }
    }
  }
}
